src/constructors.jl

```diff
@@ -111,8 +111,12 @@ One can find some circumstances where `inline=true` is faster, and other circums
 where `inline=false` is faster, so the best setting may require experimentation.
 
 `unroll` is an integer that specifies the loop unrolling factor, or a
-tuple `(4, 2)` signaling that the generated code should unroll more than
-one loop.
+tuple `(u₁, u₂) = (4, 2)` signaling that the generated code should unroll more than
+one loop. `u₁` is the unrolling factor for the innermost loop and `u₂` for the next-innermost loop,
```

Comment:
"inner most loop being unrolled" and "next inner most loop being unrolled". There may be loops that are inner with respect to these that are not getting unrolled.